### ISSUE TITLE
Baseline Creation Helpers

### ISSUE THREAD

**Issue by seandenigris** [seandenigris/Pharo-Preferences#2]
We already have `ConfigurationOf class>>#createBaselineFrom: selector`. It would be nice to have an analogous message to create a boilerplace BaselineOf from scratch, maybe from a package?

MetacelloToolBox has some relevant functionality. E.g.:
```smalltalk
MetacelloToolBox 
	createBaselineOf: 'cPanelLive' 	requiredProjects: #()
	packages: #('CPanel')
	repositories: #()
	dependencies: #()
	groups: #()
```

**Comment by seandenigris** [comment]
MetacelloToolBox has some relevant functionality. E.g.:
```smalltalk
MetacelloToolBox 
	createBaselineOf: 'cPanelLive' 	requiredProjects: #()
	packages: #('CPanel')
	repositories: #()
	dependencies: #()
	groups: #()
```

**Comment by seandenigris** [comment]
Fixed in 0b4af06
